apps/web/src/server/app/app.controller.js

```diff
@@ -36,22 +36,20 @@ export function viewHomepage(request, response, next) {
 	// The user belongs to a single group in the applications service only
 
 	if (applicationGroupIds.length === 1) {
```

Comment:
here you need to check if a use belongs to at least one of the valid `applicationGroups` and redirect them to '/applications-service'.

There s no instance where the user gets redirected to to the dashboard page anymore